controller/src/main/java/io/pravega/controller/store/ZKStoreHelper.java

```diff
@@ -55,6 +55,7 @@ public ZKStoreHelper(final CuratorFramework cf, Executor executor) {
         this.executor = executor;
         this.cache = new Cache();
         //Listen for any zookeeper connectivity error and relinquish leadership.
```

Comment:
Please change this comment to "Listen of zookeeper connection state changes" and add it just before `client.getConnectionStateListenable().addListener`

Reply:
Done